docs/source/guide/explanation/algorithms/object_detection/object_detection.rst

```diff
@@ -1,17 +1,116 @@
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------+---------------------+-----------------+
+| `Custom_Object_Detection_Gen3_SSD <https://github.com/openvinotoolkit/training_extensions/blob/develop/otx/algorithms/detection/configs/detection/mobilenetv2_ssd/template.yaml>`_    | SSD   | 9.4                 | 7.6             |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------+---------------------+-----------------+
+| `Custom_Object_Detection_Gen3_ATSS <https://github.com/openvinotoolkit/training_extensions/blob/develop/otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml>`_  | ATSS  | 20.6                | 9.1             |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------+---------------------+-----------------+
+
+`ATSS <https://arxiv.org/abs/1912.02424>`_ is a good medium-range model that works well and fast in most cases. 
+`SSD <https://arxiv.org/abs/1512.02325>`_ and `YOLOX <https://arxiv.org/abs/2107.08430>`_ are light models, that a perfect for the fastest inference on low-power hardware.
+YOLOX achieved the same accuracy as SSD, and even outperforms its inference on CPU 1.5 times, but requires 3 times more time for training, which is even more than for ATSS. So if you have resources for a long training, you can pick the YOLOX model.
+
+Besides this, we support public backbones from `torchvision <https://pytorch.org/vision/stable/index.html>`_, `pytorchcv <https://github.com/osmr/imgclsmob>`_, `mmcls <https://github.com/open-mmlab/mmclassification>`_ and `OpenVino Model Zoo <https://github.com/openvinotoolkit/open_model_zoo>`_.
+Please, refer to the :doc:`tutorial <../../../tutorials/advanced/backbones>` how to customize models and run public backbones.
+
+To see which public backbones are available for the task, the following command can be executed:
+
+.. code-block::
+
+        $ otx find --backbone {torchvision, pytorchcv, mmcls, omz.mmcls}
+
+In the table below the mAP on some academic datasets using our :ref:`supervised pipeline <od_supervised_pipeline>` is presented.
+The results were obtained on our templates without any changes.
+For hyperparameters, please, refer to the related template.
+We trained each model with a single Nvidia GeForce RTX3090.
+
++-----------+------------+-----------+
+| Model name| COCO       | MinneApple|
++===========+============+===========+
+| YOLOX     | N/A        | N/A       |
```

Comment:
Will N/A part be added in this PR?

Reply:
Good point. For COCO I don't have results on hand, but I found results for MinneApple (although it's not a well-known dataset)

Reply:
Added Pascal VOC dataset to be trained as well

Reply:
I think we can add pretrained weights' performance for COCO or we can delete the COCO performance in this table.
I think deleting could be an option since OTX is transfer learning tool(COCO -> COCO looks weird). In addition VOC results should be contained in this table. Will you update this?